lms/static/scripts/frontend_apps/components/BookSelector.js

```diff
@@ -183,23 +181,21 @@ export default function BookSelector({
 
         {selectedBook && (
           <div
-            className="hyp-u-layout-row--center hyp-u-horizontal-spacing--2"
+            className="flex flex-row items-center space-x-2"
             data-testid="selected-book"
           >
-            <Icon name="check" classes="hyp-u-color--success" />
-            <div className="hyp-u-stretch BookSelector__title">
-              {selectedBook.title}
-            </div>
+            <Icon name="check" classes="text-success" />
```

Comment:
Where is `text-success` coming from? 'Tailwind utility classes based on the tailwind configuration', I suppose.

Reply:
I'm really glad you asked this question and please keep asking about anything that isn't immediately obvious.

`success` is a _color_ defined in the Tailwind configuration (from the shared package). Tailwind will make a bunch of utility classes available based on that color, of which `text-success` is one. As soon as I saw the generated classname, I was annoyed. The background color classnames are a little more obvious (`bg-success`, for example), but you just kind of have to know that foreground/text colors are set with classes named `text-{color}`.

I wonder if naming the colors better could help (this will require a change to the shared package but isn't a huge deal).